### PR TITLE
Add nounzip to schema

### PR DESCRIPTION
**Pull request by imobach**
It fixes [bsc#962620](https://bugzilla.suse.com/show_bug.cgi?id=962620). The `nounzip` element is introduced by perl-Bootloader.

### CODE REVIEW

File: package/yast2-bootloader.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jan 19 17:12:10 UTC 2016 - igonzalezsosa@suse.com
+
+- Add nounzip elemento to AutoYaST schema (bsc#962620)
```
Comment (kobliha) [inline package/yast2-bootloader.changes]: `elemento` did not validate :)

File: src/config/bootloader.rnc [hunk 2]
```diff
@@ -104,7 +104,8 @@ section =
     element remap { "true" | "false" }? &
     element configfile {text}? &
     element image   {text}? &
-    element description {text}?
+    element description {text}? &
+    element nounzip {text}?
```
Comment (kobliha) [inline src/config/bootloader.rnc]: Was this actually already dropped from master? I could not find it: https://github.com/search?q=nounzip+user%3Ayast&type=Code

Comment (imobach) [inline src/config/bootloader.rnc]: I think we should port this change to master according to https://github.com/openSUSE/perl-bootloader/search?utf8=%E2%9C%93&q=nounzip

**Comment by kobliha** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

File: src/config/bootloader.rnc (hunk 2 above)
Comment (jreidinger) [inline src/config/bootloader.rnc]: no, as it is grub1 only parameter, grub2 do not allow to set it

Comment (imobach) [inline src/config/bootloader.rnc]: @jreidinger Thanks for clarification.

**Pull request merged**